### ISSUE TITLE
Add explicit support to R and other languages.

### ISSUE THREAD

**Issue by FrancyJGLisboa** [timedomain-tech/open-creator#8]
Hi. Interesting project. 
However, it would be nice to add support to R. I tried to run the saved skills (R functions) and it threw an error saying it is not supported. Maybe it would be interesting to add a `code_language` parameter in the creator.create or in the .run method.

**Comment by ChuxiJ** [comment]
Nice suggestion. It will be very easy to add new language support after the v0.1.2 release. #7 

Please stay tuned. 

If you can provide more usage cases, I can better assist you!

**Comment by ChuxiJ** [comment]
@FrancyJGLisboa 

![edf3027c-3b34-4c40-9da8-99668d28ceb8](https://github.com/timedomain-tech/open-creator/assets/30956809/3b1dc335-1dff-4f7e-aa42-271fde890d6e)

This is due to bugs in code interpreter.

issue fixed: #15 

Install and test

```shell
git clone https://github.com/timedomain-tech/open-creator.git
```

```shell
cd open-creator
```

```shell
git checkout v0.1.3
git pull
```

```shell
pip install .
```

try my examples

```python
from creator import create

skill = create(request="create a graph with numbers from 1 to 10 on both the x and y axis in r language, we have installed ggplot2")

```

```python
skill.save()
```

```python
skill.show()
skill.show_code()
```

```python
skill.run("run this skill")
```

**Comment by FrancyJGLisboa** [comment]
Thank you!
